app/shared/directives/activityLog/activityLog.css

```diff
@@ -0,0 +1,74 @@
+.activity-log__dark-overlay {
+  /* A dark translucent div that covers the whole screen */
+  position: fixed;
+  z-index: 100;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #000000;
+  opacity: 0.5;
+}
+
+.activity-log__container {
+  /* A centered div above the overlay with a box shadow. */
+  z-index: 100;
+  position: fixed;
+  width: 50%; /* Default */
+
+  /* Center the dialog */
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  -webkit-transform: translate(-50%, -50%);
+  -moz-transform: translate(-50%, -50%);
+  box-shadow: 0 0 8px 2px rgba(0, 0, 0, 0.15);
+}
```

Comment:
Depending on how the element is contained, centering can be done easily with flexbox, but it's up to you.